src/game.cpp

```diff
@@ -2742,8 +2747,8 @@ void Game::playerAcceptTrade(uint32_t playerId)
 		return;
 	}
 
-	if (!canThrowObjectTo(tradePartner->getPosition(), player->getPosition())) {
-		player->sendCancelMessage(RETURNVALUE_CREATUREISNOTREACHABLE);
+	if (!canThrowObjectTo(tradePartner->getPosition(), player->getPosition()), true, true) {
```

Comment:
you broke this. forgot to remove ")"?

Reply:
autocompletion broke when I was putting the second boolean and both me and the compiler overlooked that 😅